modules/Microsoft.Logic/workflows/readme.md

```diff
@@ -7,16 +7,17 @@ This module deploys a Logic App resource.
 - [Resource types](#Resource-types)
 - [Parameters](#Parameters)
 - [Outputs](#Outputs)
+- [Cross-referenced modules](#Cross-referenced-modules)
 - [Deployment examples](#Deployment-examples)
 
 ## Resource types
 
-| Resource Type | API Version |
-| :-- | :-- |
-| `Microsoft.Authorization/locks` | [2017-04-01](https://docs.microsoft.com/en-us/azure/templates/Microsoft.Authorization/2017-04-01/locks) |
+| Resource Type                             | API Version                                                                                                                       |
+| :---------------------------------------- | :-------------------------------------------------------------------------------------------------------------------------------- |
```

Comment:
It is very likely this autoformatting will fail the pipeline. Please undo all occurences  (`-` & spaces)

Reply:
@MrMCake 
Made the requested changes, let me know if its correct